mifosng-android/src/main/res/values-hi/strings.xml

```diff
@@ -0,0 +1,544 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">मीफोसXड्रायड</string>
```

Comment:
@therajanmaurya, I have made the required changes. Please let me know if I have to make any further changes, I'll be happy to do the needful.